packages/extension-api-explorer/src/ExtensionApiExplorer.tsx

```diff
@@ -115,12 +120,19 @@ export const ExtensionApiExplorer: FC = () => {
     if (sdk && !specs) loadSpecs().catch((err) => console.error(err))
   }, [specs, sdk])
 
+  const extensionEnvAdaptor = new ExtensionEnvAdaptor(getExtensionSDK())
+
   return (
-    <RunItProvider sdk={sdk} configurator={configurator} basePath="">
-      <>
-        {specs && <ApiExplorer specs={specs} />}
-        {!specs && 'Loading API specifications from Looker ...'}
-      </>
-    </RunItProvider>
+    <Provider store={store}>
+      <RunItProvider sdk={sdk} configurator={configurator} basePath="">
+        <>
+          {specs ? (
+            <ApiExplorer specs={specs} envAdaptor={extensionEnvAdaptor} />
+          ) : (
+            'Loading API specifications from Looker ...'
```

Comment:
Minor todo .. we should remove `from Looker` at some point to make this general.